src/app/core/services/youtube.search.spec.ts

```diff
@@ -30,7 +37,7 @@ describe('Youtube Search Service', () => {
     });
   });
 
-  // instantiation through framework injection
+  //  // instantiation through framework injection
```

Comment:
There seems to be a duplicate comment indicator `//`. Please remove the extra `//` to maintain clean and readable code.